src/get-started/test-service.md

```diff
@@ -100,13 +99,13 @@ To perform testing, we first need to create a network emulation – the eponymou
 Since `TestKit` will be used by all tests, it is natural to move its
 constructor to a separate function:
 
+<!-- markdownlint-disable MD013 -->
```

Comment:
I'd suggest to format `TestKit::for_rust_service` in several lines instead; presently, it creates a horizontal scrollbar. Also, the `<!-- markdownlint-enable -->` comment is the reason of the failing CI build (`mkdocs` inserts a bogus `</p>` tag after it for some reason).